app/experimenter/experiments/api/v6/serializers.py

```diff
@@ -0,0 +1,105 @@
+import json
+
+from rest_framework import serializers
+
+from experimenter.experiments.models import (
+    NimbusBranch,
+    NimbusBucketRange,
+    NimbusExperiment,
+)
+
+
+class NimbusBucketRangeSerializer(serializers.ModelSerializer):
+    randomizationUnit = serializers.ReadOnlyField(
+        source="isolation_group.randomization_unit"
+    )
+    namespace = serializers.ReadOnlyField(source="isolation_group.namespace")
+    total = serializers.ReadOnlyField(source="isolation_group.total")
+
+    class Meta:
+        model = NimbusBucketRange
+        fields = (
+            "randomizationUnit",
+            "namespace",
+            "start",
+            "count",
+            "total",
+        )
+
+
+class NimbusBranchSerializer(serializers.ModelSerializer):
+    feature = serializers.SerializerMethodField()
+
+    class Meta:
+        model = NimbusBranch
+        fields = ("slug", "ratio", "feature")
+
+    def get_feature(self, obj):
+        return {
+            "featureId": obj.experiment.feature_config.slug,
+            "enabled": obj.enabled,
+            "value": obj.value,
+        }
+
+
+class NimbusExperimentSerializer(serializers.ModelSerializer):
+    id = serializers.ReadOnlyField(source="slug")
```

Comment:
These aren't stable because the UX lets the user change them at will. Is that still an appropriate output identifier?

Reply:
Yep, the experiment locks and is no longer editable when it is shipped to remote settings, so it becomes stable at that point.

Reply:
Unless its rejected, in which case it can be edited again.

Reply:
Yeah so the way we handled this in rapid was to omit experiments in `DRAFT` and `REVIEW` states from the API altogether, so that it only appears in the API when its in its locked state.  We could do that here too?

Reply:
Well, there's the question of whether we should allow the user to edit the slug at all. If we're going to use this as an identifier (which it appears we are), I'd prefer the user have nothing to do with it. e.g. we slugify it on the server and that's what it is from then on.

Reply:
Yeah that's what we did with legacy, slugs were never exposed in the UI and only autogenerated once at creation time and not editable after that.  But we found that in some cases users wanted to have explicit control over them so the theory in Nimbus was to expose them in the UI and then just follow the pattern of locking the experiment once it moves out of Experimenter.